Cesium3DTilesSelection/src/Tileset.cpp

```diff
@@ -1509,6 +1541,15 @@ void Tileset::_unloadCachedTiles(double timeBudget) noexcept {
 
 void Tileset::_markTileVisited(Tile& tile) noexcept {
   this->_loadedTiles.insertAtTail(tile);
+  // Don't clear the children of this tile next frame
+  this->_externalTilesPendingClear.remove(&tile);
+  if (tile.getState() == TileLoadState::Unloaded &&
+      !tile.getChildren().empty()) {
+    // We were going to clear this tile's children, but it's still in use, so we
+    // should restore it to Done instead.
+    tile.setState(TileLoadState::Done);
+    tile.setContentShouldContinueUpdating(false);
+  }
```

Comment:
This is the cause of the missing tiles. `tile.getState() == TileLoadState::Unloaded && !tile.getChildren().empty()` will be true for a very large number of tiles before the first time they're loaded. Setting them to `Done` will ensure they never get loaded. Do you need to check that it's also in `_externalTilesPendingClear` maybe?